docs/docker-image-details.md

```diff
@@ -82,6 +82,9 @@ You can use this kind of image to test the catalog of Apache Hive.
 
 Changelog
 
+- gravitino-ci-hive:0.1.9
+  - Let user `hive` be able to access MySQL.
```

Comment:
As there was no new image release after the last update, version 0.1.9 simply clears the cache compared to 0.1.8.

So here should be: `remove cache after installing packages`